app/views/providers/means/policy_disregards/show.html.erb

```diff
@@ -13,17 +13,16 @@
         <p class="govuk-error-message" id="savings-amount-cash-error">
           <span class="govuk-visually-hidden">Error: </span><%= @form.errors[:england_infected_blood_support].first %></p>
       <% end %>
-    <div class="deselect-group" data-deselect-ctrl="#policy-disregards-none-selected-true-field">
-      <% Providers::PolicyDisregardsForm::SINGLE_VALUE_ATTRIBUTES.each do |checkbox| %>
-        <%= form.govuk_check_box checkbox, true, "", multiple: false, link_errors: true, label: { text: t(".#{checkbox}") } %>
-      <% end %>
-    </div>
-
-    <%= form.govuk_radio_divider %>
-    <%= form.govuk_check_box :none_selected, true, "", multiple: false, label: { text: t(".none_of_these") } %>
+      <div class="deselect-group" data-deselect-ctrl="#policy-disregards-none-selected-true-field">
+        <% Providers::PolicyDisregardsForm::SINGLE_VALUE_ATTRIBUTES.each do |checkbox| %>
+          <%= form.govuk_check_box checkbox, true, "", multiple: false, link_errors: true, label: { text: t(".#{checkbox}") } %>
+        <% end %>
```

Comment:
Form builder guidance: "the first checkbox link_errors must be set to true"

Line 18, would would set true on all checkboxes.

Consider call `.each_with_index` on `PolicyDisregardsForm::SINGLE_VALUE_ATTRIBUTES`
then using `link_errors: index.zero?` to set the first checkbox link_errors to true.

Reply:
This is the old page, I think I just changed the indentation so it was more readable. I guess it was never implemented correctly at the time 🤷‍♀️ I could update it but this page will be removed/unused pretty soon anyway!